Detectors/PHOS/calib/src/phos-calib-workflow.cxx

```diff
@@ -33,6 +33,10 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
   workflowOptions.push_back(ConfigParamSpec{"energy", o2::framework::VariantType::Bool, false, {"collect tree for E calib"}});
   workflowOptions.push_back(ConfigParamSpec{"badmap", o2::framework::VariantType::Bool, false, {"do bad map calculation"}});
 
+  workflowOptions.push_back(ConfigParamSpec{"output-dir", o2::framework::VariantType::String, "./", {"ROOT files output directory"}});
+  workflowOptions.push_back(ConfigParamSpec{"meta-output-dir", o2::framework::VariantType::String, "./", {"metafile output directory"}});
```

Comment:
```suggestion
  workflowOptions.push_back(ConfigParamSpec{"meta-output-dir", o2::framework::VariantType::String, "/dev/null", {"metafile output directory"}});
```

other workflows also define it so IIRC they need to be uniform in their default value.